README.md

```diff
@@ -4,20 +4,73 @@
 
 This repository contains example data and setup instructions for the Redis University course [RU203: Querying, Indexing, and Full-Text Search with Redis](https://university.redislabs.com/courses/ru203/).
 
-Follow the steps in this document to get ready to run the example queries from the course.
+To take this course, you'll first need do the following:
 
-## Getting help
+1. Get RediSearch and load the sample data
+1. Build the RediSearch indexes by running the index creation commands
 
 Need help getting started? Stop by the [Redis University Discord server](https://discord.gg/wYQJsk5c4A).
 
-## Getting RediSearch
+To get RediSearch, we recommend either launching it from Docker (for beginners) or building it from source and running the locally (more advanced).
 
-You can get a free, cloud-hosted instance of Redis with RediSearch by creating a [Redis Cloud Essentials](https://redislabs.com/try-free/) account.
+## Option 1: Run RediSearch locally with Docker (Best option for beginners)
 
-To run RediSearch locally, you have a couple of options:
+First, make sure you have [Docker installed](https://docs.docker.com/get-docker/).
 
-* [Run the RediSearch Docker container](https://redislabs.com/blog/getting-started-with-redisearch-2-0/)
-* [Build RediSearch from source](https://oss.redislabs.com/redisearch/Quick_Start/?_gl=1*v4bgp*_gcl_aw*R0NMLjE2MDc3MjIwMDIuQ2owS0NRaUF6c3otQlJDQ0FSSXNBTm90RmdNUnl6NXZQMk9lSzYyTXR2OXZJY0JRNXlZUDRBMnNSTXVrc2RQVDdfZkhtRG9JdThsUmdBZ2FBa2g3RUFMd193Y0I.#building_and_running_from_source)
+### 1. Start Redis
+
+Next, run the following command from your terminal:
+
+```
+docker run -it --rm --name redis-search-2 -p 6379:6379  redislabs/redisearch:2.0.5
```

Comment:
Maybe we should have them `-d` to daemonize it? Or else in step 2 just mention they need to open a new terminal.